elements/pfe-icon/package.json

```diff
@@ -9,33 +9,24 @@
     "access": "public"
   },
   "description": "Icon element for PatternFly Elements",
-  "keywords": [
-    "web-components",
-    "html"
-  ],
   "repository": {
     "type": "git",
-    "url": "github:patternfly/patternfly-elements",
-    "directory": "elements/pfe-icon"
+    "url": ""
```

Comment:
Why regress the url paths and main locations?

Reply:
I'll pull in those fields from the previous package.json.